ui/src/main/webapp/components/sites/site.spec.js

```diff
@@ -1,26 +1,10 @@
-/*global test, mount, expect */
+/*global test, shallow, expect */
 import React from 'react'
 import Site from './Site'
 
 test('site renders correctly', () => {
-  const wrapper = mount(
-    <Site name='example' connected={true} onClick={() => {}} />
+  const wrapper = shallow(
+    <Site name='example' content='www.example.com' id='1234' />
```

Comment:
❓ Do we have any other automated testing right now?

Reply:
Nope :D